gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -612,11 +612,12 @@ L.corpse_too_far_away = "Leiche zu weit weg zum Untersuchen."
 L.corpse_binoculars = "Drücke [{key}] um Leiche mit Fernglas zu untersuchen."
 L.corpse_searched_by_detective = "Diese Leiche wurde von einem Detektiv untersucht"
 
-L.target_disg = "gestarnt"
+L.target_disg = "Getarnt"
 L.target_unid = "Unidentifizierter Körper"
 
 L.target_credits = "Durchsuche, um ungenutzte Credits zu erhalten."
 
+L.target_switch_weapon = "Drücke [Knopf] um mit aktueller Waffe zu tauschen"
```

Comment:
"Knopf" shouldn't be statically haha :D But that's nothing on your side.

Reply:
I saw that too and wasn't sure whether or not Tim intended it to be that way haha

Reply:
yeah, we should talk about this before the 0.6 release. Currently there is no mention of the key in the text, it only references the keysymbol next to it. We could change it however

Reply:
@TimGoll resolve it if you created an issue, pls :)